packages/engineer/test/fixtures/engine-feature-roots/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "@fixture/engine-single-feature",
+  "version": "14.3.1",
```

Comment:
Bump to 16.0.0

Reply:
I assume all test fixtures, right?

Reply:
I think yes, also fixed copypasted package name 😅